rlmolecule/molecule/molecule_state.py

```diff
@@ -1,37 +1,13 @@
-from typing import Iterable, List, Optional
+from typing import Sequence
 
 from rdkit.Chem import Mol, MolToSmiles
 
+from molecule_game.molecule_config import MoleculeConfig
```

Comment:
Any reason you're moving the MoleculeConfig class? my hope was to eventually delete that `molecule_game` folder, after moving things into the `rlmolecule` directory. I was thinking that stable radical-specific code would live in the `examples/` directory, while anything more generally applicable to molecules would live in `rlmolecule.molecule`.

Reply:
Nope! I was trying to use any config as if it were a simple object as that it would be easy to swap out with other implementations.